worknenjoy/gitpay
# Issue#800

## Description

> Fixing overlap of chip on task status when the content is big and the screen size is small.

## Changes

- task-table.js

## Issue

> Closes #800

## Impacted Area

> Task list page

## Steps to test

Steps needed to reproduce the scenario from this change to validate if the pull request solve this issue

- Create / log user
- Import a task with a big project name (ex.: https://github.com/asmyshlyaev177/react-horizontal-scrolling-menu/issues/82)
- Go to task list page
- Resize the screen to verify if the chip that contains the project name fit to the table and not overlap the task status chip

## Before
>
![image](https://user-images.githubusercontent.com/21283066/119428573-803cd300-bce3-11eb-93dc-1060a2464b06.png)
![image](https://user-images.githubusercontent.com/21283066/119428616-95b1fd00-bce3-11eb-8f86-6b491f6187b6.png)
![image](https://user-images.githubusercontent.com/21283066/119428641-a5c9dc80-bce3-11eb-8a82-a27f965f53c1.png)


## After
> 
![print 01](https://user-images.githubusercontent.com/21283066/119428682-c09c5100-bce3-11eb-9c22-45ce43f190c5.png)
![print 02](https://user-images.githubusercontent.com/21283066/119428684-c134e780-bce3-11eb-84ef-c83b5a168fa7.png)
![print 03](https://user-images.githubusercontent.com/21283066/119428686-c1cd7e00-bce3-11eb-977a-43d22a27f5a9.png)
